design/metastore.md

```diff
@@ -22,81 +23,79 @@ Deliver manageability of metastore information alongside the object-store data a
 
 ### Design Diagram
 
-[lakeFS Metastore](diagrams/metastore.png)
+![lakeFS Metastore](diagrams/metastore.png)
 
 In this diagram:
-1. Spark/Hive/Trino - data and metadata clients
-1. lakeFS Metastore Thrift Proxy - act as the client's metastore by implementing the Hive Metastore Thrift protocol
-1. lakeFS - Include the new following components:
-   - Additional operations on OpenAPI that will 
-   - Metadata catalog that will wrap Graveler to serve metadata like we do for the object store.
+1. Spark/Hive/Trino - data and metadata clients (connects using Thrift endpoint)
+1. lakeFS Metastore Thrift Proxy - acts as the client's metastore by implementing the Hive Metastore Thrift protocol
+1. lakeFS - Includes the metatore implementation by:
+   - Additional operations on OpenAPI that will serve the metastore information and operations like we have for objects.
+   - Metadata catalog that will use Graveler and store metastore information.
```

Comment:
Again, I would much have the metastore on top of lakeFS rather than inside it.  The metastore data are really quite tiny, and I am not sure why we would need to go full Graveler on them, rather than just keeping a JSON-equivalent structure inside a file.